org-roam.el

```diff
@@ -426,8 +426,8 @@ https://github.com/kaushalmodi/ox-hugo/blob/a80b250987bc770600c424a10b3bca6ff728
       ret)))
 
 (defmacro org-roam--with-temp-buffer (&rest body)
-  "Call \"with-temp-buffer\", propagating \"org-roam-directory\"
-to the temp buffer."
+  "Call `with-temp-buffer`, propagating `org-roam-directory` to
```

Comment:
the closing quote is a single quote, you should see a difference in the syntax highlight.